rt/features/logging/src/main/java/org/apache/cxf/ext/logging/AbstractLoggingInterceptor.java

```diff
@@ -102,8 +115,13 @@ public void createExchangeId(Message message) {
         }
     }
 
-    protected String transform(final String originalLogString) {
+    protected String transform(final Message message, final String originalLogString) {
         return originalLogString;
     }
 
+    protected String maskSensitiveElements(final Message message, String originalLogString) {
+        return Optional.of(maskSensitiveHelper)
```

Comment:
Why Optional here when maskSensitiveHelper is declared as final?

Reply:
Optional makes no sense anymore, stays from time, when maskSensitiveHelper was instantiated only by setter and wasn't final. Removed.